ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Books of the Old Testament</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 30</idno>
+                        <altIdentifier>
+                        <idno facs="https://archivesetmanuscrits.bnf.fr/ark:/12148/cc99487b">BnF Éthiopien d'Abbadie 35</idno>
+                          </altIdentifier>
+                        </msIdentifier>
+
+
+                          <msContents>
+                          <msItem xml:id="ms_i1">
+                             <locus from="3ra" to="47rc"/>
+                             <title type="complete" ref="LIT1340EnochE"/>
+                             <incipit xml:lang="gez"><hi rend="rub">ዘሄኖክ᎓ ነቢይ᎓ ቃለ᎓ በረከት᎓</hi> ዘከመ᎓ ባረከ᎓
+                               ኅሩ<hi rend="rub">ያነ᎓ ወጻድቃነ᎓ እለ᎓ ሀ</hi>ለዉ᎓ ይኩኑ᎓ በዕለተ᎓ ም<hi rend="rub">
+                               ንዳቤ᎓ ለአሰስሎ᎓ ኲሎ᎓</hi> እኩያን᎓ ወረሲዓን፨</incipit>
+                             <explicit xml:lang="gez">ወየሐውሩ᎓ እሙንቱሂ᎓ በኀበ᎓ ተጽሕፈ᎓ ሎሙ᎓ መዋዕል᎓ ወአዝማን፨</explicit>
+                             <note>Book of Henok. Division into parts is indicated by the word <foreign>ክፍ</foreign> or
+                             <foreign xml:lang="gez">ክፍል</foreign>, rubricated, without any number.
+                             </note>
+                                </msItem>
+
+                                <msItem xml:id="ms_i2">
+                                  <locus from="47ra" to="88vc"/>
+                                  <title type="complete" ref="LIT5832BooksSalomon"/>
+                                  <note>Books of Salomon.</note>
```

Comment:
I believe that the `<note>` here and elsewhere containing the title is redundant (and might then add to a cluttered view on the app), since you provided the correct title ID everywhere

Reply:
I leave it for a while for myself, for better orientation.